instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/rum/ActivityTrackingTest.kt

```diff
@@ -29,9 +29,18 @@ internal abstract class ActivityTrackingTest :
 
         instrumentation.waitForIdleSync()
 
-        // one for the Application start action
+        expectedEvents.add(ExpectedApplicationStart())
```

Comment:
should we align the naming used? Seems we have `ApplicationStart` and `ApplicationLaunch` at the same time.

UPD: seems we are a bit late, both definitions already live in the codebase.

Reply:
I'll fix so it's "ApplicationStartEvent" and "ApplicaitonLaunchView" which I know doesn't make things that much clearer but at least it will be consistent.